notebooks/237-typo-detector/237-typo-detector.ipynb

```diff
@@ -0,0 +1,537 @@
+    "MODEL_DIR = \"model2/\"\n",
+    "\n",
+    "MODEL_DIR = f\"{MODEL_DIR}\"\n",
+    "\n",
+    "onnx_model_path = Path(MODEL_DIR) / onnx_model\n",
+    "\n",
+    "print(onnx_model_path)\n",
+    "\n",
+    "dummy_model_input = tokenizer(\"This is a sample\", return_tensors=\"pt\")\n",
+    "\n",
+    "torch.onnx.export(\n",
+    "    model,\n",
+    "    tuple(dummy_model_input.values()),\n",
+    "    f=onnx_model_path,\n",
+    "    input_names=['input_ids', 'attention_mask'],\n",
+    "    output_names=['logits'],\n",
+    "    dynamic_axes={'input_ids': {0: 'batch_size', 1: 'sequence'},\n",
+    "                  'attention_mask': {0: 'batch_size', 1: 'sequence'},\n",
+    "                  'logits': {0: 'batch_size', 1: 'sequence'}},\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Model Optimizer\n",
+    "\n",
+    "[Model Optimizer](https://docs.openvino.ai/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) is a cross-platform command-line tool that facilitates the transition between training and deployment environments, performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices.\n",
+    "Model Optimizer converts the model to the OpenVINO Intermediate Representation format (IR), which you can infer later with [OpenVINO runtime](https://docs.openvino.ai/latest/openvino_docs_OV_UG_OV_Runtime_User_Guide.html#doxid-openvino-docs-o-v-u-g-o-v-runtime-user-guide)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "optimizer_command = f'mo \\\n",
+    "    --input_model {onnx_model_path} \\\n",
+    "    --output_dir {MODEL_DIR} \\\n",
+    "    --model_name {model_id} \\\n",
+    "    --input input_ids,attention_mask \\\n",
+    "    '\n",
+    "! $optimizer_command"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "OpenVINO™ Runtime uses the [Infer Request](https://docs.openvino.ai/latest/openvino_docs_OV_UG_Infer_request.html) mechanism which allows running models on different devices in asynchronous or synchronous manners. The model graph is sent as an argument to the OpenVINO API and an inference request is created. The default inference mode is AUTO but it can be changed according to requirements and hardware available. You can explore the different inference modes and their usage [in documentation.](https://docs.openvino.ai/latest/openvino_docs_Runtime_Inference_Modes_Overview.html)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from openvino.runtime import Core\n",
+    "\n",
+    "ie = Core()\n",
+    "ir_model_xml = str((Path(MODEL_DIR) / model_id).with_suffix(\".xml\"))\n",
+    "compiled_model = ie.compile_model(ir_model_xml)\n",
+    "infer_request = compiled_model.create_infer_request()"
```

Comment:
Don't need to create infer request for OpenVINO running inference. Examples are like this: 
compiled_model = ie.compile_model(ov_model, device)
results = compiled_model.output(0)